### PR TITLE
TF-2596 Mailto invalid with several recipients

### PR DESCRIPTION
**Pull request by dab246**
## Issue

#2596 

## Resolved

https://github.com/linagora/tmail-flutter/assets/80730648/77e2f4a9-db76-49a9-b0ee-3641744a4e2e

### CODE REVIEW

**Review by chibenwa — COMMENTED** [review]
Please document this route and the exact syntax somewhere in `/doc`.

We could maybe detail the different URI schemes that can be relied on to interact with Twake Mail?

**Comment by dab246** [comment]
> Please document this route and the exact syntax somewhere in `/doc`.
> 
> We could maybe detail the different URI schemes that can be relied on to interact with Twake Mail?

Updated

File: docs/adr/0036-mailto-uri-chemes-to-interact-twake-mail.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Decision
+
+Summary of URI schemes that can interact with Twake Mail:
+
+- `/mailto?uri=user@example.com`
+- `/mailto/?uri=mailto:user@example.com&subject=TwakeMail&body=HelloWorld`
+- `/mailto/?uri=mailto:user1@example.com,user2@example.com,user3@example.com&subject=TwakeMail&body=HelloWorld`
+
+The URI scheme may or may not be encoded
```
Comment (hoangdat) [inline docs/adr/0036-mailto-uri-chemes-to-interact-twake-mail.md:21]: please give example about url encoded too

Comment (dab246) [reply docs/adr/0036-mailto-uri-chemes-to-interact-twake-mail.md]: done

**Review by hoangdat — APPROVED** [review]

**Pull request merged**